android/app/src/main/java/app/organicmaps/car/screens/NavigationScreen.java

```diff
@@ -93,6 +94,28 @@ public void onStopNavigation()
     mRoutingController.cancel();
   }
 
+  /**
+   * To verify your app's navigation functionality when you submit it to the Google Play Store, your app must implement
+   * the NavigationManagerCallback.onAutoDriveEnabled callback. When this callback is called, your app must simulate
+   * navigation to the chosen destination when the user begins navigation. Your app can exit this mode whenever
+   * the lifecycle of the current Session reaches the Lifecycle.Event.ON_DESTROY state.
```

Comment:
`Session reaches the Lifecycle.Event.ON_DESTROY`

That means many routes could be tested by starting auto-drive mode only once.
We may have problems with the current implementation of NavigationScreen.
NavigationManager has to be moved to the CarAppSession.

This may also fix the following issue: https://github.com/organicmaps/organicmaps/issues/7167
Right now when you leave NavigationScreen (onDestroy) navigation is stopped. This shouldn't happen when opening another app but we have this issue, so, it is likely related.

Reply:
I suggest attempting to submit this implementation first. A full refactoring of the navigation flow will likely be time-consuming, and we are currently unsure about the testing process.